### ISSUE TITLE
[expo-notifications] getExpoPushTokenAsync() & getDevicePushTokenAsync() not resolving

### ISSUE THREAD

**Issue by arberK** [expo/expo#7921]
I wanted to integrate the expo-notifications through unimodules in my bare react-native app. 

I followed all the instructions to install it. I can ask permissions with the module. But the methods get*PushTokenAsync() are not getting resolved.

I enabled Push Notifications in the Capabilities section in XCode as well as in AppStoreConnect. 
I also generated a CSR Certificate in AppleStoreConnect (I don't know if this one is necessary). 

Am I missing something or is this a real bug? Does anyone else face the same problem? 

I only tested in iOS 13.3.1

"react": 16.12.0
"react-native": 0.61.5
"expo-notifications":0.1.1
"react-native-unimodules": 0.9.0

**Comment by byCedric** [comment]
Hi @arberK! Thanks for reporting this. Unfortunately, without a [small repro](https://stackoverflow.com/help/minimal-reproducible-example) I'm not sure what's going wrong. Have you checked out [Brent's example for notifications in bare](https://forums.expo.io/t/expo-notifications-getexpopushtokenasync/35740/2)? Maybe that could help you figure out what's missing from your project 😄 

Hope it helps!

**Comment by kinu01** [comment]
Experiencing the same issue too.

**Comment by kinu01** [comment]
@arberK Were you able to resolve it?

**Comment by thorbenprimke** [comment]
I ran into the same problem. I had to reboot the device as mentioned on [SO](https://stackoverflow.com/questions/58264338/not-getting-apns-device-token-on-ios-13) and that fixed it. 

I was switching in between different scheme / profiles and perhaps something is cached such that it doesn't know about the push notification capability such that it passes the permission check but hangs on getting device / push token. Anyway, reboot fixed it for me.
